### PR TITLE
Timeline Events (Device+Sim) and bug fixes

### PR DESCRIPTION
**Pull request by karthik-zensark**

### CODE REVIEW

**Comment by SeanCondon** [comment]
retest this please

**Review by SeanCondon — APPROVED** [review]
I am still seeing problems, but I'd like to merge this PR and have you fix them in a follow up one.

**Comment by karthik-zensark** [comment]
Noted Sean. Will br resolved in the next PR.

Thank you.

Regards,
Karthik Gaddam
+918686400545
***@***.******@***.***>
________________________________

**Comment by karthik-zensark** [comment]
> I am still seeing problems, but I'd like to merge this PR and have you fix them in a follow up one.

Noted Sean. Will be resolved in the next PR.

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -6,5 +6,5 @@
 
 export const environment = {
   production: true,
-  baseUrl: '/',
+  baseUrl: 'https://chronos-dev.onlab.us/',
```
[the remark this answers is not in the document]
Comment (SeanCondon) [reply src/environments/environment.prod.ts:9]: Just change it to `'/'` as it was before please

[the remark this answers is not in the document]
Comment (karthik-zensark) [reply src/environments/environment.prod.ts:9]: Changed it '/' and commented the old one for now as it being used elsewhere.

**Pull request merged**